praw/models/reddit/subreddit.py

```diff
@@ -1586,8 +1586,13 @@ def conversations(self, after=None, limit=25, other_subreddits=None,
 
         response = self.subreddit._reddit.get(
             API_PATH['modmail_conversations'], params=params)
-        return [self(conversation_id)
-                for conversation_id in response['conversationIds']]
+        conversations = []
+        for conversation_id in response['conversationIds']:
+            data = {'conversation': response['conversations'][conversation_id]}
+            conversations.append(
+                ModmailConversation.parse(data, self.subreddit._reddit,
+                                          convert_objects=False))
```

Comment:
The response only contains the most recent message. So, converting the objects would mean creating a `ModmailConversation` object that's inconsistent with the one that would be individually fetched from the API.